lib/optimizely/event_builder.rb

```diff
@@ -74,9 +76,9 @@ def get_common_params(user_id, attributes)
       visitor_attributes = []
 
       attributes&.keys&.each do |attribute_key|
-        # Omit null attribute values
+        # Omit invalid attribute values.
```

Comment:
correct comment is "Omit attribute values that are not supported by the log endpoint."
we need to check attribute key is string or not?